packages/crashlytics/ios/RNFBCrashlytics/RNFBCrashlyticsModule.m

```diff
@@ -45,13 +44,13 @@ + (BOOL)requiresMainQueueSetup {
 
 RCT_EXPORT_METHOD(crash) {
   if ([RNFBCrashlyticsInitProvider isCrashlyticsCollectionEnabled]) {
-    [[Crashlytics sharedInstance] crash];
+    assert(NO);
```

Comment:
I double checked this, it is the recommended method now in upstream docs

Reply:
Ref: https://firebase.google.com/docs/crashlytics/upgrade-sdk?platform=ios#the_crash_and_throwexception_methods_are_removed